website/blog/2023-07-02-databend-weekly-100.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "This Week in Databend #100"
+date: 2023-07-02
+slug: 2023-07-02-databend-weekly
+tags: [weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: dantengsky
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: everpcpc
+  - name: JackTan25
+  - name: leiysky
+  - name: lichuang
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+  - name: ZhiHanZ
+  - name: zhyass
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> Having trouble connecting your root user to Databend? Read [Docs | Configuring Admin Users](https://databend.rs/doc/sql-clients/admin-users) to learn how to set up correctly.
```

Comment:
For security reasons, the Root user is no longer available out of the box. You must configure it before use. Learn more at https://databend.rs/doc/sql-clients/admin-users.